Examples/BrandGame/BrandGame/OpenTelemetry/Export/Log/StdoutLogExporter.swift

```diff
@@ -40,10 +40,10 @@ class StdoutLogExporter: LogRecordExporter {
         return .success
     }
 
-    func forceFlush() -> OpenTelemetrySdk.ExportResult {
+    func forceFlush(explicitTimeout: TimeInterval?) -> OpenTelemetrySdk.ExportResult {
         return .success
     }
 
-    func shutdown() { }
+    func shutdown(explicitTimeout: TimeInterval?) { }
```

Comment:
This was done because `LogRecordExporter` changed?
Probably this is something we should communicate in the release notes of `6.5.0`. Mostly because this changes that were necessary to be done on the BrandGame app, will probably be also necessary for any customer using a custom `LogRecordExporter`